src/main/java/emt/item/focus/ItemMaintenanceFocus.java

```diff
@@ -48,7 +48,9 @@ public ItemStack onFocusRightClick(ItemStack itemStack, World world, EntityPlaye
         ItemWandCasting wand = (ItemWandCasting) itemStack.getItem();
 
         if (!world.isRemote) {
-            if (player.isSneaking()) {
+            if (!player.isSneaking()) {
+                return itemStack;
+            } else if (world.getTileEntity(mop.blockX, mop.blockY, mop.blockZ) instanceof BaseMetaTileEntity) {
```

Comment:
Duplicated `getTileEntity` calls. Make a variable and reuse it.
Also `else` is not needed here